src/functions/frontend/toDelta.m

```diff
@@ -1,7 +1,7 @@
-function delta_out = toDelta(rct_obj)
+function delta_out = toDelta(rct_obj, varargin)
```

Comment:
`varargin` is at most going to be one element, correct? If so, it'd be best to replace with a more meaningful name, like `delta_map`

Reply:
I wrote the mapping as optional functionality. Fully naming it means that it would be a required variable

Reply:
No, you can have optional variables with different names.  See, for example, the constructor for [DeltaSlti](https://github.com/iqcToolbox/iqcToolbox/blob/master/src/classes/Delta/DeltaSlti.m):
```
DeltaSlti('d')
DeltaSlti('d', 1, -2, 3)
```